src/test/java/no/nav/tag/sykefravarsstatistikk/api/ApiTest.java

```diff
@@ -55,6 +55,7 @@ public void sammenligning__skal_returnere_riktig_objekt() throws Exception {
                 "    \"prosent\": 5.1," +
                 "    \"erMaskert\": false" +
                 "  }," +
+                "  \"bransje\":null," +
```

Comment:
Skal vi vise _bransje_ i tillegg til _næring_ når virksomheten er inn i bransjeprogrammet eller skal det bare se ut som / erstatte _næring_ ? 
Ser i utgangspunktet ikke noe grunn til at vi skiller de to.

Reply:
De skal ha forskjellig tekst i frontend, så jeg måtte finne en hensiktsmessig måte å skille dem på. Kunne hatt et flagg som f.eks. "erBransje" eller noe, men da tenkte jeg det var like greit å være konsekvent